xbmc/powermanagement/PowerManager.cpp

```diff
@@ -62,6 +62,7 @@ using namespace ANNOUNCEMENT;
 CPowerManager g_powerManager;
 
 CPowerManager::CPowerManager()
+  : m_lastPlayer("")
```

Comment:
not needed. std::string initializes itself to "". further we are moving away from contructor initand to it in hrader files (c++11)

Reply:
> further we are moving away from contructor init and to it in header files (c++11)

 This is not part of our coding rules. I consider this personal taste. Personally, I stick with "old school" ctor init lists. There is no worse and better here, both approaches have advantages and downsides.

Reply:
we've collectively agreed that it's better to adopt c++11 features. Our coding rules should be updated for c++11.

Also I think the advantages outweigh the downsides in this case, because it scales better when adding multiple constructors.

Also this discussion is moot because `std::string` doesn't need to be explicitly initialized :)